### PR TITLE
test(textarea): update with new approach

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

Apply new `url` approach in  tests for:
`Experimental:`
`TextArea` (timing: was `5:00`- > `01:36`),
`TextBox` (timing: was `03:34` -> `00:45`),
`TextBox Actions` (timing: `00:14`),
`TextBox-multiple` (timing: was `03:13` -> `00:50`).

### Current behaviour

Currently we are passing all parameters via `knobs`.

### Checklist

- [ ] Screenshots are included in the PR
- [ ] Carbon implementation and Design System documentation are congruent
- [ ] All themes are supported
- [x] Commits follow our style guide
- [ ] Unit tests added or updated
- [x] Cypress automation tests added or updated
- [ ] Storybook added or updated
- [ ] Typescript `d.ts` file added or updated

### CODE REVIEW

File: cypress/features/regression/experimental/textarea.feature [hunk 1]
```diff
@@ -1,243 +1,225 @@
...
-
-  Background: Open Experimental Textarea component page
-    Given I open "Experimental Textarea" component page
+  I want to check Experimental Textarea component properties
 
   @positive
-  Scenario: Enable expandable checkbox for a Textarea component
-    When I check expandable checkbox
+  Scenario Outline: Enable expandable checkbox for a Textarea component
+    Given I open default "Experimental-Textarea" component in noIFrame with "textarea" json from "experimental" using "expandableEnabled" object name
```
Comment (KatarzynaQA) [inline cypress/features/regression/experimental/textarea.feature:6]:
```suggestion
    Given I open default "Experimental-Textarea" component in noIFrame with "textarea" json from "experimental" using "expandable" object name
```

File: cypress/fixtures/experimental/textarea.json [hunk 2]
```diff
@@ -0,0 +1,201 @@
+{ 
+  "expandableEnabled": {
```
Comment (KatarzynaQA) [inline cypress/fixtures/experimental/textarea.json:2]:
```suggestion
  "expandable": {
```

File: cypress/fixtures/experimental/textarea.json [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    "expandable": true
+  },
+  "expandableDisabled": {
```
Comment (KatarzynaQA) [inline cypress/fixtures/experimental/textarea.json:5]:
```suggestion
  "expandableFalse": {
```

File: cypress/features/regression/experimental/textarea.feature [hunk 4]
```diff
@@ -1,243 +1,225 @@
...
-  Scenario Outline: Disable warnOverLimit checkbox for a Textarea component and allow to input more characters than allowed
-    When I set characterLimit to "<limit>"
-      And I check warnOverLimit checkbox
-      And I uncheck warnOverLimit checkbox
-      And I check enforceCharacterLimit checkbox
-      And I uncheck enforceCharacterLimit checkbox
-      And I input <text> into Textarea
-    Then Textarea component has no warnOverLimit and used characters <characters> of <limit>
+  Scenario Outline: Enable and disable expandable checkbox for a Textarea component
+    Given I open default "Experimental-Textarea" component in noIFrame with "textarea" json from "experimental" using "expandableDisabled" object name
```
Comment (KatarzynaQA) [inline cypress/features/regression/experimental/textarea.feature:15]:
```suggestion
    Given I open default "Experimental-Textarea" component in noIFrame with "textarea" json from "experimental" using "expandableFalse" object name
```

File: cypress/features/regression/experimental/textarea.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/features/regression/experimental/textarea.feature]: updated

File: cypress/fixtures/experimental/textarea.json (hunk 2 above)
Comment (DlgSHi) [reply cypress/fixtures/experimental/textarea.json]: updated

File: cypress/fixtures/experimental/textarea.json (hunk 3 above)
Comment (DlgSHi) [reply cypress/fixtures/experimental/textarea.json]: updated

File: cypress/features/regression/experimental/textarea.feature (hunk 4 above)
Comment (DlgSHi) [reply cypress/features/regression/experimental/textarea.feature]: updated

**Pull request merged**